tests/update_test.py

```diff
@@ -81,7 +81,7 @@ def test_onupdate_is_triggered(self):
         args = args[1]
         self.assertEqual(expected_stmt, stmt)
         self.assertEqual(40, args[0])
-        dt = datetime.strptime(args[1], '%Y-%m-%dT%H:%M:%S.%fZ')
+        dt = datetime.strptime(args[1], '%Y-%m-%dT%H:%M:%S.%f')
```

Comment:
Can not use the `%z` suffix here. Otherwise:

```python
ValueError: time data '2024-06-25T13:48:33.295461' does not match format '%Y-%m-%dT%H:%M:%S.%f%z'
```